gravitee-apim-distribution/pom.xml

```diff
@@ -105,10 +105,14 @@
         <gravitee-fetcher-http.version>1.12.0</gravitee-fetcher-http.version>
         <gravitee-repository-elasticsearch.version>3.11.0</gravitee-repository-elasticsearch.version>
         <!-- Gateway Only -->
+        <gravitee-notifier-email.version>1.3.2</gravitee-notifier-email.version>
+        <gravitee-notifier-slack.version>1.2.1</gravitee-notifier-slack.version>
+        <gravitee-notifier-webhook.version>1.1.0</gravitee-notifier-webhook.version>
```

Comment:
not sure about this

Reply:
APIM 3.15 use bom > 2.1,
So we can use notifier-email 1.4.1

For the others, it's ok.